### DISCUSSION TITLE
Welcome to MFFS Discussions!

### DISCUSSION THREAD

**Discussion by GaeaKat** [KatCodesMods/mffs#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tifasnow** [comment]
Hey everyone!

I'm [Tifa](https://www.tifapup.com), @tifasnow here on GitHub. I'm going to be working on the Documentation and Support side of things, taking over from @aimeesunflower who previously helped on the 1.12.x versions.
